gulpfile.js

```diff
@@ -177,7 +177,7 @@ gulp.task('fonts', () => {
  * Creates a sourcemap.
  ******************************************************************************/
 
-gulp.task('js', ['js:hint', 'js:cs', 'browserify-tests'], () => {
+gulp.task('js', ['js:hint', 'js:cs'], () => {
```

Comment:
@davidcmoulton correct me if I'm wrong, but it seems `js:hint` and `js:cs` do not take into account the contents of `test/`. If that's true, we don't need to run them separately in the `ci` image (which is the only one actually containing the tests), which would be great.

Reply:
@giorgiosironi yes, `js:hint` and `js:cs` do not run over the `test/` directory (although whether they should is another question).

Reply:
I'll keep the current behavior, checks could be run there if needed, but they would be different tasks so that they can be separated